.github/workflows/specs.yml

```diff
@@ -3,24 +3,20 @@ on: [push, pull_request]
 jobs:
   tests:
     strategy:
+      fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        ruby: [2.7]
     runs-on: ${{ matrix.os }}
     steps:
       - uses: ruby/setup-ruby@v1
         with:
-          ruby-version: ${{ matrix.ruby }}
+          ruby-version: 2.7
       - uses: actions/checkout@v2
-      - name: Specs
-        run: |
-          gem install tf -v '>=0.4.1'
-          hash -r
-          ./install
-          git submodule init
```

Comment:
We do not need this anymore?

Reply:
81b56ae4747aa7f2d3056fa92a7df1ad18095338
>  git submodule commands are no longer needed, there are no submodules in the rvm repo anymore

Reply:
I remember we talked about it, but dont remember if we executed this change or not?

Reply:
Yes it's done, there is no `.gitmodules` at the root of the repo anymore